src/client/Microsoft.Identity.Client/ILongRunningWebApi.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Methods for long running processes in web APIs
+    /// </summary>
+    public interface ILongRunningWebApi
```

Comment:
Should probably mentiontion the exceptions that will be thrown by these apis. Similar to the comment on `AcquireTokenSilent`